.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+keys
```

Comment:
nice job adding sensitive files here!
I would also add:
.DS_Store
Since you're on mac